src/asound/bluealsa-ctl.c

```diff
@@ -709,6 +902,8 @@ static int bluealsa_get_integer_info(snd_ctl_ext_t *ext, snd_ctl_ext_key_t key,
 		*imin = 0;
 		*istep = 1;
 		break;
+	default:
```

Comment:
Keep `switch` without `default` (in other places as well). Of course there are cases where switch without a default might become unreadable. But I think that we are safe here.